src/base/locale/es/LC_MESSAGES/messages.po

```diff
@@ -1,138 +1,177 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
+# Translators:
+# Jacob <signup@futuresoup.com>, 2016
+# PB Project <frank+pbproject@openplans.org>, 2016
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
```

Comment:
This is really odd--it was translated in my branch. Looks like the keys differ significantly. Did we get rid of the translation because it was outdated?